server/src/test/java/io/crate/integrationtests/LogicalReplicationITest.java

```diff
@@ -216,6 +221,29 @@ public void test_subscribing_to_single_publication() throws Exception {
                                                      "2\n"));
     }
 
+    @Test
+    public void test_create_subscription_system_tables_filled() throws Exception {
+        executeOnPublisher("CREATE TABLE doc.t1 (id INT) WITH(" + defaultTableSettings() + ")");
+        executeOnPublisher("CREATE TABLE doc.t2 (id INT) WITH(" + defaultTableSettings() + ")");
+
+        // If isForAllTables is true, list argument is used only to grant DQL (to make subscribe work) but publication is not aware of tables in the moment of creation.
```

Comment:
Does this comment make sense here if the publication is created with concrete tables? ;)

Reply:
with concrete tables output will be the same but I wanted to run this case through more tricky "internally" workflow, so that tables are resolved only after subscription is created - perhaps phrasing not clear - should I remove if it's confusing?

Reply:
Not sure if I understand you correctly ;) In this test the publication is created with a concrete list of tables and `forAllTables = false`. Is this a mistake and you wanted to test `forAllTables = true` instead? then this comment would make sense to me and is worth leaving to avoid confusion.

Reply:
Oh, yep, it's a mistake, I pushed version with forAllTables = true. Thanks!